packages/user-extension/src/index.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+/**
+ * @packageDocumentation
+ * @module user-extension
+ */
+
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { IStateDB } from '@jupyterlab/statedb';
+import {
+  ICurrentUser,
+  IUserMenu,
+  RendererUserMenu,
+  User,
+  UserMenu
+} from '@jupyterlab/user';
+import { Menu, MenuBar, Widget } from '@lumino/widgets';
+
+/**
+ * A namespace for command IDs.
+ */
+export namespace CommandIDs {
+  export const settings = '@jupyterlab/user-extension:settings:open';
+}
+
+/**
+ *
+ */
+const userPlugin: JupyterFrontEndPlugin<User> = {
```

Comment:
Plugin template should match its contract:

```suggestion
const userPlugin: JupyterFrontEndPlugin<ICurrentUser> = {
```

Reply:
Should this be the interface `IUser` instead of `ICurrentUser` and the same on the next comment?